source/content/restore-environment-backup.md

```diff
@@ -87,3 +87,37 @@ This is a destructive process. If you're not comfortable with this technique, us
 5. Deploy the change from Dev up to Test and Live.
 
 For more information, see [git reset](https://git-scm.com/docs/git-reset).
+
+## Restoring the LIVE environment
+The restore process removes any recent content or changes applied to your site, since the date of the backup file you will use. Doing so in LIVE means you will lose them forever with no way to recover.
+
+If there is still a need to do this in the LIVE environment, we recommend doing the following steps:
+- [Lock the environment first](/docs/security/)
+- [Run a backup](/docs/backups/) of LIVE, in order to have a copy of files and the database for later use
+- Run the restore while the site is still locked
+
+Doing these steps will allow you to later recreate any new content manually. It also resticts access while the restore process is still running, which is a good practice. 
+
+## Restoring Large Sites
+Large sites that have more than 100GB files can take too long to restore and are likely to fail. We recommend running the steps above for a safer process, or do use the separate [backup archive files as mentioned above](/docs/restore-environment-backup/#restore-an-environment-from-another-environments-backup).
```

Comment:
Which steps above? Aren't these all steps to restore?

Reply:
above points to the steps that involve locking LIVE temporarily etc.